pkg/config/config.go

```diff
@@ -39,6 +39,10 @@ type Config struct {
 	// MaxArrayValues is the maximum number of array items that the commands
 	// print, locals, args and vars should read (in verbose mode).
 	MaxArrayValues *int `yaml:"max-array-values,omitempty"`
+
+	// If ShowLocationExpr is true whatis will print the DWARF location
+	// expression for its argument.
+	ShowLocationExpr bool `yaml:"show-location-expr"`
```

Comment:
It was useful to debug problems in the compiler side. I think knowing where a variable is stored is useful in general.

Reply:
I'm hesitant to add features to Delve that are useful only for us an potentially not so much for the end user. This same feature could be done w/ `readelf` and some grepping from anyone hacking on Delve that needs to actually see this information.

How much of a pain would it be to split this out into a separate PR and have a discussion there, so we can pull in the rest of these changes?

Also, if we add this, it might be worth adding a flag to `whatis` to optionally print this information during a live debug session w/o having to set global (or session) config.